docs/data_encryption.md

```diff
@@ -0,0 +1,253 @@
+Encrypting your data
+====================
+
+This example is shown using etcd, but any of the supported backends *should* work.
+
+- [PGP Keypair](#local-testing-keypair)
+  - [Sample Keypair](#local-testing-keypair-sample)
+  - [Generating Your Own Keypair](#local-testing-keypair-generating)
+- [Encrypting & Storing Your Data](#local-testing-saving)
+  - [With GPG](#local-testing-saving-gpg)
+  - [With crypt](#local-testing-saving-crypt)
+- [Verifying Encrypted Data](#local-testing-verify)
+- [Putting It All Together, Using With Confd](#local-testing-confd)
+
+<a name="local-testing-keypair"/>
```

Comment:
Ah, I see the Table of contents above. Let's remove it, as no other documentation in confd has this.